skill/tests/mock_alice.py

```diff
@@ -0,0 +1,316 @@
+from fluentcheck import Check
+
+
+def chained(fn):
+    def new(*args, **kwargs):
+        fn(*args, **kwargs)
+        return args[0]
+
+    return new
```

Comment:
Прикольный декоратор 👍️

Можно еще для красоты и понимания что это декторатор добавить стандартную штуку:
```suggestion
from functools import wraps

def chained(fn):
    @wraps(fn)
    def new(*args, **kwargs):
        fn(*args, **kwargs)
        return args[0]

    return new
```